lmdeploy/lite/apis/calibrate.py

```diff
@@ -118,7 +119,11 @@ def calibrate(model: str,
               calib_samples: int = 128,
               calib_seqlen: int = 2048,
               work_dir: str = './work_dir',
-              device: str = 'cuda') -> None:
+              device: str = 'cuda',
+              w_bits: int = 4,
+              w_group_size: int = 128,
+              search_scale: bool = False,
+              batch_size: bool = 1) -> None:
```

Comment:
typo: bool -> int
Is `1` a good default value?

Reply:
1 is for BC breaking, the previous design is for low memory users.